test/PlaygroundTransform/async_let.swift

```diff
@@ -28,25 +42,25 @@ async let fac4 = factorial(4)
 print(await fac4)
 
 //             return await x * factorial(x - 1)
-// CHECK:      [22:{{[[:digit:]]+}}-22:{{[[:digit:]]+}}] __builtin_log[='1']
```

Comment:
Would it make sense to use wildcards for the line numbers here also?  It looks as if that's what the other tests do.

I'm kind of on-the-fence but perhaps it would make sense to have a single focused test on correct lines and columns, and all the others using wildcards?  This one looks particular to async-let?

Reply:
It might make sense, but I purposefully left the test logic unchanged.  I didn't want to make any assumptions about specific tests, as part of this change, until we have a chance to look at them individually. I like to keep the changes isolated to the purpose of the change.